lwreg/test_lwreg.py

```diff
@@ -951,9 +953,34 @@ class TestRegisterConformersPSQL(TestRegisterConformers):
 
     def setUp(self):
         super(TestRegisterConformersPSQL, self).setUp()
+        getlogin = lambda: pwd.getpwuid(os.getuid())[0]
         self._config['dbname'] = 'lwreg_tests'
         self._config['dbtype'] = 'postgresql'
+        self._config['password'] = 'testpw'
+        self._config['user'] = getlogin()
 
+    def testNoSecretsInRegistrationMetadata(self):
+        """Make sure initdb is not storing any secrets."""
+        utils._initdb(config=self._config, confirm=True)
+        with utils.connect(self._config).cursor() as cursor:
+            cursor.execute('select * from registration_metadata;')
+            keys = set(v[0] for v in cursor.fetchall())
+            assert not any(v in keys for v in ('user', 'password'))
```

Comment:
we're using unittest to do testing, please use methods of the test instance instead of naked asserts

Reply:
Changed it accordingly